lib/Rest/VstMessage.h

```diff
@@ -1,109 +0,0 @@
-////////////////////////////////////////////////////////////////////////////////
```

Comment:
is this file now just empty, or has it been deleted by this PR?

Reply:
deleted